website/docs/d/autoscaling_group.html.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: "aws"
+page_title: "AWS: aws_autoscaling_group"
+sidebar_current: "docs-aws-datasource-autoscaling-group"
```

Comment:
This documentation page is missing a sidebar link in `website/aws.erb` in the data sources section.